### DISCUSSION TITLE
achievements

### DISCUSSION THREAD

**Discussion by moki** [community/community#18154] [Other Feature Feedback, Questions, & Ideas]
I guess gamification is a natural addition to any product at this point.

If you will go in that direction consider thinking about ways for people to share them, configuring visibility(profile display), easy way to see what kind of achievements they can work towards, seasonal achievements.

**Comment by owieth** [comment]
totally agree!

**Comment by confessore** [comment]
i wish achievements meant something to non-tech people. cool, you’re a bronze pull shark and an arctic vault contributor but what does that mean to someone who does not understand this platform or its people?

**Reply by TDroogers** [reply]
I will most definitely send a printscreen of pull shark to my friends, however they will not perceive it the same way you and I do.

**Comment by olegkron** [comment]
IMO, making all possible achievements visible kinda shifts user's goals from "Cool, new one! Back to work." to "Man, I have to make X more commits this month to get this next achievement." A bit exaggerated, but you get the point. It may slightly affect user's behaviour on the platform. The undisclosed list always has something to surprise you with, but always gets you back to work :)

**Reply by moki** [reply]
Well, you kinda described the whole point of the gamification.

i mean it is a dark pattern by nature

**Comment by wallaceleonel** [comment]
Acho sensacional esses novos emblemas ahaha

**Comment by Ysrbolles** [comment]
totally agree! 2

**Reply by Ericles-Porty** [reply]
totally agree! 3

**Comment by epistax1s** [comment]
We are moving towards steam. I like.
